File: AppGranja/app/src/main/java/com/example/appgranja/SheepsFragment.java
package com.example.appgranja;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.TextView;

public class SheepsFragment extends Fragment {

    private Button sumar,restar;
    private TextView contador_text;
    private int contador = 0;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_sheeps, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        contador_text = getView().findViewById(R.id.Contador);
        sumar = getView().findViewById(R.id.button_sum);
        sumar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                contador++;
                contador_text.setText("Contador: "+contador);
            }
        });


    }


    /*@Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        sumar = (Button) getView().findViewById(R.id.button_sum);
        sumar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                contador++;
                contador_text.setText("Contador"+contador);
            }
        });

    }*/


}

File: KAppGranja/app/src/main/java/com/example/kappgranja/ui/management/SheepsFragment.kt
package com.example.kappgranja.ui.management

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.kappgranja.R

class SheepsFragment : Fragment(){
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        val view = inflater.inflate(R.layout.fragment_sheeps, container, false)
        return view
    }

}